src/actions/actions.ts

```diff
@@ -0,0 +1,66 @@
+import { Webhooks } from '../webhooks/webhooks';
+import { ResponsePayload } from './interfaces/response-payload';
+
+export class Actions {
+  private webhooks: Webhooks;
+
+  constructor(webhooks: Webhooks) {
+    this.webhooks = webhooks;
+  }
+
+  private get computeSignature() {
+    return this.webhooks.computeSignature.bind(this.webhooks);
+  }
+
+  get verifyHeader() {
+    return this.webhooks.verifyHeader.bind(this.webhooks);
+  }
+
+  async signResponse({
+    type,
+    verdict,
+    errorMessage,
+    secret,
+  }: {
+    type: 'authentication' | 'user_registration';
+    verdict: 'allow' | 'deny';
```

Comment:
I think our API interface says `Allow` or `Deny`, though we also accept the lowercase versions. We should try to be consistent. Do we prefer the capitalized version or the lowercase version?

Reply:
Let's go with capitalized here 👍